packages/backend/server/src/plugins/indexer/__tests__/init-es.sh

```
@@ -0,0 +1,1505 @@
+#  ------------ elasticsearch: es ------------
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)